src/components/SQForm/SQFormMultiValue.js

```diff
@@ -0,0 +1,271 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Grid, Chip, TextField, makeStyles} from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import {VariableSizeList} from 'react-window';
+import {useField, useFormikContext} from 'formik';
+import {useForm} from './useForm';
+
+// MUI uses px, a numeric value is needed for calculations
+const LISTBOX_PADDING = 8; // px
+
+const OuterElementContext = React.createContext({});
+
+const OuterElementType = React.forwardRef((props, ref) => {
+  const outerProps = React.useContext(OuterElementContext);
+  return <div ref={ref} {...props} {...outerProps} />;
+});
+
+function renderRow({data, index, style}) {
+  return React.cloneElement(data[index], {
+    style: {
+      ...style,
+      top: style.top + LISTBOX_PADDING
+    }
+  });
+}
+
+// Adapter for react-window
+const ListboxVirtualizedComponent = React.forwardRef(
+  function ListboxVirtualizedComponent(props, ref) {
+    const {children, ...listboxProps} = props;
+    const LIST_MAX_VIEWABLE_ITEMS = 8;
+    const LIST_OVERSCAN_COUNT = 5;
+    const items = React.Children.toArray(children);
+    const ITEM_COUNT = items.length;
+    const ITEM_SIZE = 36;
```

Comment:
Should these constants live with the one on line 10?

Reply:
These constants are only used in this closure so they were scoped to this closure